nixtla/nixtla_client.py

```diff
@@ -945,8 +978,10 @@ def forecast(
             "level": level,
             "finetune_steps": finetune_steps,
             "finetune_loss": finetune_loss,
+            "feature_contributions": feature_contributions,
```

Comment:
Asking for contributions without providing `X` doesn't currently raise an error on the API, but it should
```suggestion
            "feature_contributions": feature_contributions and X is not None,
```